kong.conf.default

```diff
@@ -173,17 +173,49 @@
                                  #   It runs DB-less and receives configuration
                                  #   updates from a control plane node.
 
+#cluster_mtls = shared           #
+                                 # Valid values to this setting are:
+                                 #
+                                 # - `shared`: use a shared certificate/key
+                                 #   pair specified with the `cluster_cert`
+                                 #   and `cluster_cert_key` settings.
+                                 #   Note that CP and DP nodes have to present
+                                 #   the same certificate to establish mTLS
+                                 #   connections.
+                                 # - `pki`: use `cluster_ca_cert` and
+                                 #   `cluster_server_name` for verification,
+                                 #   these are different certificates for each
+                                 #   DP node, but issued by a cluster-wide
+                                 #   common CA certificate: `cluster_ca_cert`
+
 #cluster_cert =                  # Filename of the cluster certificate to use
                                  # when establishing secure communication
                                  # between control and data plane nodes.
                                  # You can use the `kong hybrid` command to
                                  # generate the certificate/key pair.
+                                 # Should be a different certificate for each
+                                 # DP node.
 
 #cluster_cert_key =              # Filename of the cluster certificate key to
                                  # use when establishing secure communication
                                  # between control and data plane nodes.
                                  # You can use the `kong hybrid` command to
                                  # generate the certificate/key pair.
+                                 # Should be a different certificate for each
+                                 # DP node.
+
+#cluster_ca_cert =               # The trusted CA certificate file in PEM
+                                 # format used to verify the `cluster_cert`.
+                                 # Required if `cluster_mtls` is set to `pki`.
```

Comment:
```suggestion
                                 # Ignored if `cluster_mtls` is not set to `pki`.
```

Might be a more accurate description because we don't use this value in `shared` mode.